docs/sdk_developers/examples.md

```
@@ -24,6 +24,23 @@ python examples/name_of_file.py
+
+Some example scripts (notably those related to Ethereum / EVM functionality) require **optional dependencies** that are not installed by default.
+
+If you encounter import errors related to Ethereum libraries, install the ETH extra before running those examples.
+
+#### Using uv (recommended)
+
+```bash
+uv sync --dev --extra eth
+```
+
+#### Using pip
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)